clean-blog/templates/base.html

```diff
@@ -59,16 +59,7 @@
         {% endif %}
 
         <!-- Custom Fonts -->
-        <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
-        <link href='http://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
-        <link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
-
-        <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-        <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-        <!--[if lt IE 9]>
-            <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-            <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-        <![endif]-->
+        <link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/fonts/open-sans.css" rel="stylesheet">
```

Comment:
My browser cannot find the file. Shouldn't the line be changed as follows?
```html
<link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/fonts/open-sans/open-sans.css" rel="stylesheet">
```

Reply:
yes, indeed